### PR TITLE
Add support for Synchronization UpdatedTransactions to Ponto Connect

### PR DESCRIPTION
**Pull request by bramj**

### CODE REVIEW

File: src/main/java/com/ibanity/apis/client/products/ponto_connect/services/impl/TransactionServiceImpl.java [hunk 1]
```diff
@@ -46,17 +47,25 @@ public IbanityCollection<Transaction> list(TransactionsReadQuery transactionsRea
             pagingSpec = IbanityPagingSpec.DEFAULT_PAGING_SPEC;
         }
 
-        URI uri = buildUri(getUrl(transactionsReadQuery.getAccountId()), pagingSpec);
+        URI uri = buildUri(getUrl(transactionsReadQuery.getAccountId(), transactionsReadQuery.getSynchronizationId()), pagingSpec);
```
Comment (damienbr) [inline src/main/java/com/ibanity/apis/client/products/ponto_connect/services/impl/TransactionServiceImpl.java:50]: Maybe worth creating a new method called listUpdatedTransactions or listUpdatedTransactionsForSynchronization like you named it in the unit test + specific query parameter,  so it's clear for the developer that it's a subset of transactions.

File: src/main/java/com/ibanity/apis/client/products/ponto_connect/models/read/TransactionsReadQuery.java [hunk 2]
```diff
@@ -19,6 +19,7 @@ public class TransactionsReadQuery {
 
     private String accessToken;
     private UUID accountId;
+    private UUID synchronizationId;
```
Comment (damienbr) [inline src/main/java/com/ibanity/apis/client/products/ponto_connect/models/read/TransactionsReadQuery.java:22]: Maybe worth extracting the use case for updatedTransactions

File: src/main/java/com/ibanity/apis/client/products/ponto_connect/services/impl/TransactionServiceImpl.java (hunk 1 above)
Comment (bramj) [reply src/main/java/com/ibanity/apis/client/products/ponto_connect/services/impl/TransactionServiceImpl.java]: Change made. Is this what you had in mind? If it's good like this I'll make the same change on XS2A for consistency.

Comment (damienbr) [reply src/main/java/com/ibanity/apis/client/products/ponto_connect/services/impl/TransactionServiceImpl.java]: Yes cool :+1:

**Pull request merged**